src/components/calcite-handle/calcite-handle.tsx

```diff
@@ -21,7 +21,7 @@ export class CalciteHandle {
   /**
    * Value for the button title attribute
    */
-  @Prop({ mutable: true, reflect: true }) textTitle = "handle";
+  @Prop({ reflect: true }) textTitle = "handle";
```

Comment:
this isn't mutable right?

Reply:
The rule detected it isn't being mutated in the component, so bye-bye. 👋